packages/react-async/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+typescript@^3.6.2:
+  version "3.6.2"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.6.2.tgz#105b0f1934119dde543ac8eb71af3a91009efe54"
+  integrity sha512-lmQ4L+J6mnu3xweP8+rOrUwzmN+MRAj7TgtJtDaXE5PMyX2kCrklhg3rvOsOIfNeAWMQWO2F1GPc1kMD2vLAfw==
```

Comment:
Lockfiles should not be committed.

Reply:
Are you 100% sure? If we don't commit lockfiles, no matter how much we pin dependencies in package.json, two CI builds of the same commit with a day in-between will most definitely use different packages.

Reply:
PS: this will never impact people using react-async, only make sure all people working on it have the same version of dependencies.

Reply:
For this file it shouldn't matter anyway, since the dependency on Typescript in the pacakge will be removed, once the next Version of Pika is released.